src/main/resources/config.yml

```diff
@@ -65,6 +65,7 @@ factions:
   allowNeutrality: false
   factionlessFactionName: Factionless
   factionHomeTeleportDelay: 5
+  maxMembers: 25
```

Comment:
Perhaps we should default this to -1 so that servers currently running the plugin do not experience a change in behaviour when they upgrade?

Reply:
Changed to -1!